buildconfig/CMake/MSVCSetup.cmake

```diff
@@ -128,18 +128,23 @@ else ()
   set (PARAVIEW_PYTHON_PATHS "" )
 endif ()
 
+set(MSVC_BIN_DIR ${PROJECT_BINARY_DIR}/bin/$<$<CONFIG:Release>:Release>$<$<CONFIG:Debug>:Debug>)
```

Comment:
I'm probably missing something here. Can we not do `set(MSVC_BIN_DIR ${PROJECT_BINARY_DIR}/bin/$<CONFIG>`. The `RelWithDebInfo` configuration is occasionally useful and this would the support that?

Reply:
:man_facepalming: yeah that would be sensible, currently it's akin to me doing

```
if (foo == "Debug"){
   bar = "Debug"
}
```